app/models/subscription/responses/SubscriptionInfo.scala

```diff
@@ -25,7 +25,13 @@ final case class SubscriptionInfo(id: String,
                                   gbUser: Boolean,
                                   tradingName: Option[String],
                                   primaryContact: Contact,
-                                  secondaryContact: Option[Contact])
+                                  secondaryContact: Option[Contact]) {
+
+  lazy val primaryContactName: String = primaryContact match {
+    case ic: IndividualContact => ic.individual.firstName + " " + ic.individual.lastName
```

Comment:
is it ok do 
  case ic: IndividualContact => s"${ic.individual.firstName}  ${ic.individual.lastName}"

Reply:
Updated